### DISCUSSION TITLE
which executable for Macos Arm based chips for c++ bindings?

### DISCUSSION THREAD

**Discussion by the-code-samurai-97** [isl-org/Open3D#6574] [Q&A]
i see there are darwin but not for arm based !

**Comment by ssheorey** [comment]
darwin is the macOS kernel (either x86_64 or arm64).

Here are the links from www.open3d.org or github releases

- [C++ Devel package](https://github.com/isl-org/Open3D/releases/download/v0.18.0/open3d-devel-darwin-arm64-0.18.0.tar.xz)
- [Viewer app](https://github.com/isl-org/Open3D/releases/download/v0.18.0/open3d-app-macosx-12_0-arm64-0.18.0.zip)
